src/client/feed/feedReducer.js

```diff
@@ -0,0 +1,12 @@
+import { UPDATE_FEED_RESULTS } from '../search/SearchActions';
+
+const feed = (state = { 'posts': [] }, action) => {
+  switch (action.type) {
+    case UPDATE_FEED_RESULTS:
+      return { 'posts': action.data };

```

Comment:
I'm guessing you just want to get something in that works for now and expand on things like is_fetching and RECIEVE_FEED_RESULTS in a different branch ?